requirements-dev.txt

```diff
@@ -5,3 +5,17 @@ constructs>=10.0.0
 
 pydantic
 python-dotenv
+
+
+aiobotocore==2.5.0
+botocore==1.29.76
+fsspec==2023.4.0
+kerchunk==0.1.0
+mangum>=0.10.0
+netCDF4==1.6.2
+rio-tiler==4.1.10
+rioxarray==0.14.1
+s3fs==2023.4.0
+titiler==0.11.6
+xarray==0.19.0
+zarr==2.14.2
```

Comment:
requirements-devs aimed to be used only for deployment purposes. 

if we need more `dev` dependencies for the `titiler.xarray` package we should update the pyproject.toml

Reply:
addressing this now!